cmd/release-controller/sync_verify_prow.go

```diff
@@ -62,6 +62,9 @@ func (c *Controller) ensureProwJobForReleaseTag(release *Release, verifyName str
 		return nil, terminalError{err}
 	}
 
+	if isAggregatedJob {
+		periodicConfig.Name = fmt.Sprintf("%s-%s", jobName, verifyName)
```

Comment:
where do periodicConfigs come from?  can you link to an example config.

Reply:
This is the `periodicConfig` for the aggregator job itself:
https://github.com/openshift/release/blob/23bd2851d5634cbd7912d1d4578d714881894f6f/ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml#L1540-L1624

Reply:
How do the upgrade jobs get their periodicConfig.Name which is distinct for each upgrade version without a separate entry in periodics.yaml?

Reply:
Smoke and mirrors... or the release-config:
https://github.com/openshift/release/blob/aaa3353f6f9ad4d2738ae48648744b150dda70c3/core-services/release-controller/_releases/release-ocp-4.10-ci.json#L76

Reply:
Ok, I'm finally getting there.

Can I get it as `verifyName-jobName`?  If I understand it correctly, that will let me name my verify stanzas as `aggregated-gcp-micro-upgrade` and I'll get a name `aggregated-gcp-micro-upgrade-periodic-ci-openshift-release-master-ci-4.10-e2e-gcp-upgrade`

Reply:
As currently written, the aggregator job name is actually: `release-openshift-release-analysis-aggregator`
So, adding the `verifyName` as a prefix or suffix would actually have ^^ in it's name:
 aggregated-gcp-micro-upgrade-release-openshift-release-analysis-aggregator
or
 release-openshift-release-analysis-aggregator-aggregated-gcp-micro-upgrade

Reply:
That being said, I have access to the prowJob name if you'd prefer the name to be something like you've suggested?